azure-functions/acbs-function/src/functions/acbs-amend-facility-covenant-record.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * This function is an Azure Durable sub-orchestrator function.
+ * This function cannot be invoked directly and is rather executed by an Azure durable orchestrator
+ * function.
+ *
+ * Facility Covenant Record amendment SOF
+ * **********************************
+ * This is a sub-orchestrated function invoked from it's master orchestrator.
+ * `acbs-amend-facility` is the master orchestrator for this function.
+ *
+ * Sub-Orchestration Function (SOF)
+ * --------------------------------
+ * This function cannot be invoked directly and must be invoked by a DOF.
+ *
+ * Prerequisites
+ * -------------
+ * 0. 'npm install durable-functions'
+ * 1. Durable HTTP trigger function (acbs-http)
+ * 2. DOF (acbs-amend-facility)
+ * 3. DAF (amend-facility-loan)
+ *
+ * ACBS
+ * ----
+ * This function is responsible for amending Facility Covenant Record.
+ */
+const df = require('durable-functions');
+const retryOptions = require('../../helpers/retryOptions');
+const mappings = require('../../mappings');
+
+df.app.orchestration('acbs-amend-facility-covenant-record', function* amendFacilityCovenant(context) {
+  const payload = context.df.input;
+
+  try {
+    if (!payload) {
+      throw new Error('Invalid payload provided');
+    }
+    const { facilityId, amendments } = payload;
+    const { amendment } = amendments;
+
+    const facilityCovenantRecord = yield context.df.callActivityWithRetry('update-facility-covenant', retryOptions, {
+      facilityId,
+      acbsFacilityLoanInput: mappings.facility.facilityCovenantAmend(amendment),
```

Comment:
Please follow the same coding and documentation pattern as per DAF `acbs-amend-facility-master.js` or `acbs-amend-facility-loan.js`.

Reply:
Updated to match